docs/source/conf.py

```diff
@@ -424,6 +424,7 @@
     "reset_max_memory_cached",
     "reset_peak_memory_stats",
     "set_per_process_memory_fraction",
+    "use_mem_pool",
```

Comment:
Please do not add any new entry to this file and make sure to properly document this API if it is expected to be public (which I think you're already doing btw).

Reply:
Sounds good! I'll try to figure out for real why coverage fails on my diff without an entry in this file.

Reply:
Unless you can show me the right direction :)

Reply:
Sounds good. Thanks for fixing it!